package.json

```diff
@@ -4,19 +4,25 @@
   "private": true,
   "scripts": {
     "check": "npm run lint && npm run prettier",
-    "dev": "next dev",
+    "dev": "next dev -p 3002",
```

Comment:
I'd respectfully disagree as this could cause confusion for our local testing setups. It would be nice for us to standardize on `deqm-test-server` at 3000, `bulk-export-server` at 3001 and this at 3002.

Reply:
That's fine. Could we update the README then to use 3002? 

Also the compromise I'll offer is instructions in the README for how to customize the port if you so wish: `npm run dev -- -p <your-port>`

Reply:
Added unit testing with Jest. Did not attempt to test the MeasureSelect component, mainly because we would need to provide something to mock `fetch` or use `axios` along with something like [msw](https://mswjs.io/docs/getting-started/integrate/node). This could be done in future work perhaps?